CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Additional logging of historical query flow in `UNSAFE` builds.
 - Historical query system will re-request entries if the host fails to provide them within a fixed time.
 - Renamed `ccf::CodeDigest` to `ccf:pal::PlatformAttestationMeasurement` and `get_code_id()` to `get_measurement()` (#5063).
+- Recovery procedure:
+  - The `GET /gov/recovery_share` endpoint has been replace with `GET /gov/recovery_share/{member_id}`. This can be accessed without any authentication.
+  - The `POST /gov/recovery_share` endpoint has been replaced with `POST /gov/recovery_share/{member_id}`. This request must now be signed, and will not allow purely TLS authentication.
```

Comment:
I don't understand why we are forbidding pure TLS auth here.

Reply:
If we have a request that we expect to be signed, why would we allow it to come in unsigned based on TLS cert auth? That seems like a major hole. We require proposals and ballots to be signed, this is just moving recovery shares to require the same?

Reply:
But we have no such requirement there?

Reply:
Discussed offline. Summary:

We're moving away from all member-cert based auth. Member certs are a historical artifact, members are really identified by their _key_. We authenticate with that key by requiring signatures, so this endpoint that needs to authenticate members (to prevent driveby `POST`s replacing a member's legitimate recovery share), does so with signatures _even though it doesn't store those signatures_.